synapse/models/biz.py

```diff
@@ -0,0 +1,204 @@
+import synapse.lib.module as s_module
+
+'''
+Model elements related to sales / bizdev / procurement
+'''
+
+class BizModule(s_module.CoreModule):
+    def getModelDefs(self):
+        modl = {
+            'types': (
```

Comment:
Do we want to add a biz:rfp:proposal:score type to capture proposal scores?

Reply:
This ends up being TBD for now as well because it would go on a proposal node